dear_petition/petition/constants.py

```diff
@@ -55,3 +57,16 @@
     "No Probable Cause",
     "Never To Be Served",
 )
+
+KNOWN_DISPOSITION_METHOD = {
```

Comment:
The build is failing. I think this might be part of the reason?

```suggestion
KNOWN_DISPOSITION_METHODS = {
```